analyzers/tests/SonarAnalyzer.UnitTest/Rules/CollectionEmptinessCheckingTest.cs

```diff
@@ -30,10 +30,25 @@ public class CollectionEmptinessCheckingTest
     {
         [TestMethod]
         public void CollectionEmptinessChecking_CS() =>
-            OldVerifier.VerifyAnalyzer(@"TestCases\CollectionEmptinessChecking.cs", new CS.CollectionEmptinessChecking());
+            new VerifierBuilder()
+                .AddAnalyzer(() => new CS.CollectionEmptinessChecking())
+                .AddPaths("CollectionEmptinessChecking.cs")
+                .Verify();
 
         [TestMethod]
         public void CollectionEmptinessChecking_VB() =>
-            OldVerifier.VerifyAnalyzer(@"TestCases\CollectionEmptinessChecking.vb", new VB.CollectionEmptinessChecking());
+            new VerifierBuilder()
+                .AddAnalyzer(() => new VB.CollectionEmptinessChecking())
```

Comment:
Can be simplified. And then, the `AddPaths` and `Verify` can be on a single line.
```suggestion
            new VerifierBuilder<VB.CollectionEmptinessChecking>()
```